web/client/components/mapcontrols/locate/LocateTool.jsx

```diff
@@ -14,15 +22,19 @@ const defaultOpt = {
         maxZoom: 18
     }
 };
-
+/**
+ * Common interface shared across multiple map types
+ * @prop {object} map the map object
+ * @prop {string} mapType can be openlayers, leaflet or cesium
```

Comment:
Why should we limit this to the actual implementations? There is nothing that prevents having a MapBoxGL api here.

Reply:
no intention to to limit maptypes here, just they are the one currently implemented

going to remove the list